tools/gz_msgs_generate_factory.py

```diff
@@ -131,45 +140,55 @@ def main(argv=sys.argv[1:]):
 
     args = parser.parse_args(argv)
 
-    headers = []
-    registrations = []
+    package_re = re.compile('^package (.*);$')
+    message_re = re.compile('message (.*)')
```

Comment:
Maybe use message_re = re.compile(r'message (\w*)\s?{?$') so we handle message classes linted with "{" on the same line